components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/resources/analytics-api.yaml

```diff
@@ -0,0 +1,759 @@
+swagger: '2.0'
```

Comment:
Remove this file from pull request as its in .gitignore

Reply:
Done